web/styles/portico/dev-buttons.css

```diff
@@ -0,0 +1,107 @@
+button,
+input,
+optgroup,
+select,
+textarea {
+    /* Reset default styles — borrowed from TailwindCSS Preflight */
+    font-family: inherit;
+    font-feature-settings: inherit;
+    font-variation-settings: inherit;
+    font-size: 100%;
+    font-weight: inherit;
+    line-height: inherit;
+    margin: 0;
+}
```

Comment:
I'd definitely like this page to have as little CSS as possible that's not present in the actual app; the whole point is to be a good simulation of what buttons in the app would look like.

Reply:
Moved the required properties into the `design-testing-control-element` class.